src/python/examples/streamlined_example.py

```diff
@@ -0,0 +1,24 @@
+import tritonserver
```

Comment:
Merge with the other example and add proper documentation

Reply:
Added `README.md`

Reply:
Merged `streamlined_example.py` and `inference_example.py`